src/data/migrations/20190822153830_create_resources_table.js

```diff
@@ -0,0 +1,21 @@
+exports.up = function(knex) {
+    return knex.schema.createTable("resources", resources => {
+        resources.increments()
+        resources.enu("type", ["link"]).notNullable()
```

Comment:
I wasn't sure what other types we should have here for resources, I made it an enum with only the one type we'll have to change

Reply:
Instead of `link`, `type` could be `google_doc`, `youtube_vid` and `unspecified` for now.

Reply:
Ok I changed it to include google_doc, youtube_vid and unspecified